### ISSUE TITLE
Make the Designer more accessible

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7436]
There's a good amount of accessibility work that can be done on the designer. This issue is intended to track that work.

To start, the "drag" and "remove" controls need to be accessible and actually usable. See https://github.com/microsoft/AdaptiveCards/issues/7067 for more details.

Unfortunately, the work required to do that is no small task. Pankaj and I wrote a proposal for a designer overhaul. That can be found [here](https://microsoft.sharepoint.com/teams/AdaptiveCards/_layouts/15/Doc.aspx?sourcedoc={ef9a3136-2fec-4955-b14d-0507fce791ed}&action=edit&wd=target%28Accessibility.one%7C32f608f3-b7db-425d-9f46-568eae5be2a2%2FDesigner%20Overhaul%7C9c775d7a-cecb-4433-a9f5-f521c2090582%2F%29&wdorigin=703). For simplicity, I've copied the summary below...

<details>
<summary>Designer Overhaul Summary</summary>

- There are various aspects of the designer that are only available to mouse users 
- Making these features available to keyboard users will require a refactor of the designer, which should involve 
   - Rendering every element of the card upon initialization, even if the element is initially hidden 
   - In designer mode, only the peers should be focusable whereas the elements themselves should be not-focusable 
      - Furthermore, all commands available to the peer should be invokable via keyboard (general commands like drag/drop, remove; as well as specific commands like 'Add a column') 
   - When new elements are added to the card, or when existing elements are moved around, the elements should be inserted into the html tree in such a way as to mimic their order in the card itself 
      - If this requires a complete re-rendering of the card every time such an action is invoked, then extra steps must be taken to ensure focus does not get lost 
- Aside from the preview card itself, the other panes in the designer (i.e. 'Card Elements', 'Card Structure' and 'Element Properties') also need to be improved to support keyboard users 
   - Since these panes are quite linked with each other, in particular the 'Card Structure' and 'Element Properties' panes, there should exist an easy way for keyboard users to switch between these panes 
   - All elements in these panes should be accessible/activatable by keyboard 
      - i.e. inserting an element into the preview card from the 'Card Elements' pane should be possible via keyboard instead of only via dragging onto the preview card 
      - i.e. all elements in the 'Card Structure' pane should be accessible via keyboard 

</details>

As a bonus, I strongly suggest that whoever begins to work on making the designer more accessible also keep an eye on the following queries:
- [Relevant accessibility Sev3](https://github.com/microsoft/AdaptiveCards/issues?q=is%3Aopen+is%3Aissue+label%3AArea-Accessibility+label%3AArea-Designer+label%3AA11ySev3)
- [Relevant accessibility Usable](https://github.com/microsoft/AdaptiveCards/issues?q=is%3Aopen+is%3Aissue+label%3AArea-Accessibility+label%3AArea-Designer+label%3AA11yUsable)

**Comment by anna-dingler** [comment]
## Action Items

- [ ] When adding new elements to the card or move existing elements, peers need to be inserted in the correct place in the html tree to maintain focus
- [ ] Render the entire card (even if a portion of it is hidden) on the initial load
- [ ] Incorporate the fixes from pull request #7159
- [ ] Drag control should be keyboard accessible
- [ ] [Feature] Add new key binding to switch between different panes of the designer easily
